scripts/gulpfiles/build_tasks.js

```diff
@@ -336,18 +336,47 @@ function buildDeps(done) {
     'tests/mocha'
   ];
 
-  const args = roots.map(root => `--root '${root}' `).join('');
-  execSync(
-      `closure-make-deps ${args} 2>/dev/null >'${DEPS_FILE}'`,
-      {stdio: 'inherit'});
+  function filterErrors(text) {
+    return text.split('\n')
+        .filter(
+            (line) => !/^WARNING /.test(line) ||
+                !(/Missing type declaration./.test(line) ||
+                  /illegal use of unknown JSDoc tag/.test(line)))
```

Comment:
Do you think we'll be dealing with this long enough that it would be useful to separate these out into an array?

Reply:
I sincerely hope not.  But when I find myself a fourth regexp to this conditional then it will be time to reevaluate…